packages/passport-interface/src/api/requestIssuedPCDs.ts

```diff
@@ -1,26 +1,24 @@
-import { FeedRequest, FeedResponse } from "@pcd/passport-interface";
+import { FeedRequest, FeedResponse } from "../RequestTypes";
+import { POST } from "./constants";
 
 /**
  * Given the information the server knows about the user, it is able to
  * 'issue' some pcds to the user. This function requests the set of PCDs
  * that the server wants to 'issue' to the user.
+ *
+ * Never rejects. All information encoded in the resolved response.
  */
 export async function requestIssuedPCDs(
   providerUrl: string,
   request: FeedRequest
 ): Promise<FeedResponse | undefined> {
   try {
     const url = providerUrl;
-    const response = await fetch(url, {
-      method: "POST",
-      body: JSON.stringify(request),
-      headers: {
-        "Content-Type": "application/json",
-        Accept: "application/json"
-      }
+    const res = await fetch(url, {
+      ...POST,
+      body: JSON.stringify(request)
     });
-    const issuedPCDResponse = (await response.json()) as FeedResponse;
-    return issuedPCDResponse;
+    return (await res.json()) as FeedResponse;
```

Comment:
A possible problem here is that we're just assuming that the response matches the type. If the server is running the same version of the shared API code, then it will. But third-party servers could be doing anything, including returning malformed or malicious data.